user_sessions/templatetags/user_sessions.py

```diff
@@ -40,43 +40,80 @@
 
 
 @register.filter
-def device(value):
+def platform(value):
     """
     Transform a User Agent into human readable text.
 
     Example output:
 
-    * Safari on iPhone
-    * Chrome on Windows 8.1
-    * Safari on macOS
-    * Firefox
+    * iPhone
+    * Windows 8.1
+    * macOS
     * Linux
     * None
     """
 
+    platform = None
+    for regex, name in PLATFORMS:
+        if regex.search(value):
+            platform = name
+            break
+
+    return platform
+
+
+@register.filter
+def browser(value):
+    """
+    Transform a User Agent into human readable text.
```

Comment:
Having the same docstring for all three filters will be confusing, update the docstring to reflect the return value.